src/libraries/System.Text.Json/src/System/Text/Json/ThrowHelper.Serialization.cs

```diff
@@ -194,8 +211,9 @@ public static void AddExceptionInformation(in ReadStack readStack, in Utf8JsonRe
             }
         }
 
+        [DoesNotReturn]
         [MethodImpl(MethodImplOptions.NoInlining)]
-        public static void ReThrowWithPath(in WriteStack writeStack, Exception ex)
+        public static void ReThrowWithPath(in WriteStack writeStack, Exception? ex)
```

Comment:
Same here. Remove nullability annotation?

Reply:
Same for all exception related, this is how we annotate, it is allowin null so nullable